case/src/main/kotlin/com/ritense/case/security/config/CaseHttpSecurityConfigurer.kt

```diff
@@ -70,5 +71,6 @@ class CaseHttpSecurityConfigurer : HttpSecurityConfigurer {
         private const val MANAGEMENT_CASE_LIST_COLUMN_URL = "/api/management/v1/case/{caseDefinitionName}/list-column"
         private const val MANAGEMENT_TASK_LIST_COLUMN_URL = "/api/management/v1/case/{caseDefinitionName}/task-list-column"
         private const val MANAGEMENT_TAB_URL = "/api/management/v1/case-definition/{caseDefinitionName}/tab"
+        private const val MANAGEMENT_WIDGET_TAB_URL = "/api/management/v1/case-definition/{caseDefinitionName}/widget-tab"
```

Comment:
I know we do this in more places, but when you use versioning this gets a bit iffy. Could you either rename this variable to `MANAGEMENT_WIDGET_TAB_URL_v1` or split this up into smaller parts, so the version still has to be specified?

Reply:
As discussed, the other options aren't any more readable than this.